src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/model/GroupedByCom.kt

```diff
@@ -0,0 +1,10 @@
+package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.model
+
+import io.swagger.v3.oas.annotations.media.Schema
+
+@Schema(description = "Describes an CA(OMU) case")
```

Comment:
This isn't a representation class - would just add as a private class to the service.

Reply:
We can add it to the service but not private, as it is used by public method.